sound/soc/sof/amd/acp.c

```diff
@@ -516,6 +518,15 @@ int amd_sof_acp_probe(struct snd_sof_dev *sdev)
 		return ret;
 	}
 
+	sdev->dsp_box.offset = 0;
+	sdev->dsp_box.size = BOX_SIZE_512;
+
+	sdev->host_box.offset = sdev->dsp_box.offset + sdev->dsp_box.size;
+	sdev->host_box.size = BOX_SIZE_512;
+
+	sdev->debug_box.offset = sdev->host_box.offset + sdev->host_box.size;
+	sdev->debug_box.size = BOX_SIZE_1024;
```

Comment:
I am having difficulties reconciling this change with the remaining declarations in https://github.com/thesofproject/linux/pull/3749/commits/92d7d2cf04cc9ada7662df9a73997d71f6f79c80#diff-001f04f65b0262fb5dbaf23aff7de3c44df830df4fb1d568867e198d4375b7c6R97

Reply:
> I am having difficulties reconciling this change with the remaining declarations in [92d7d2c#diff-001f04f65b0262fb5dbaf23aff7de3c44df830df4fb1d568867e198d4375b7c6R97](https://github.com/thesofproject/linux/commit/92d7d2cf04cc9ada7662df9a73997d71f6f79c80#diff-001f04f65b0262fb5dbaf23aff7de3c44df830df4fb1d568867e198d4375b7c6R97)

As mentioned in the above comment, these are the default offsets of the mailbox set in the driver.